gnd/src/main/java/com/google/android/gnd/persistence/local/room/converter/ResponseDeltasTypeConverter.java

```diff
@@ -32,18 +35,25 @@
  */
 public class ResponseDeltasTypeConverter {
 
+  private static final String KEY_FIELD_TYPE = "field type";
+  private static final String KEY_NEW_RESPONSE = "new response";
```

Comment:
Can we use short camel-cased keys without spaces here to make them more idiomatic JSON? e.g. "fieldType" and "newResponse"?

Reply:
Yes, thanks!